### DISCUSSION TITLE
Many ruleset strings cannot be localised

### DISCUSSION THREAD

**Discussion by naoei** [ppy/osu#19672] [Q&A] [answered]
- Loosely related to #19630

While trying to localize the tau ruleset, I've noticed that many fields are still `strings` rather than `LocalisableString`. Here are some that I've found:

- https://github.com/ppy/osu/blob/a83f7202ecf0a225252d878b60870310761e6c03/osu.Game/Screens/Ranking/Statistics/StatisticItem.cs#L21
- https://github.com/ppy/osu/blob/a83f7202ecf0a225252d878b60870310761e6c03/osu.Game/Rulesets/Mods/Mod.cs#L37
- https://github.com/ppy/osu/blob/a83f7202ecf0a225252d878b60870310761e6c03/osu.Game/Configuration/SettingSourceAttribute.cs#L46
- https://github.com/ppy/osu/blob/a83f7202ecf0a225252d878b60870310761e6c03/osu.Game/Screens/Play/ResumeOverlay.cs#L34

**Comment by smoogipoo** [comment, marked as answer]
Yes, these should all be `LocalisableString`. Not sure we need a discussion/issue thread for this, just do it when you need it?

**Reply by naoei** [reply]
~~it's not really an urgent matter so it can be changed whenever.~~ Misread that as "just when do you need it?".

I'll open a PR in the morning addressing this.

 I mainly just opened this discussion so that I can point out the ones that really need to be a localizable string and wanted to confirm that these are valid. Sorry if this comes off as spam.
